docs/en/advanced/Command-Line-Interface.md

```diff
@@ -1,3 +1,24 @@
+New Contents:
+This doc so far only focus on the `alluxio fs` related operations 
+and admin CLI doc only focus on `alluxio fsadmin` related operations.
+
+Other `alluxio [COMMAND other than fs/fsadmin]` is missing in our documentation.
+Some of them are really useful to our users and should be included, such as the 
+usage of `alluxio getConf [key]`. If users know `alluxio getConf --master` can get
+the current alluxio cluster configuration, that would be useful.
+
+The commands include extensions, format, formatMaster, formatWorker, bootstrapConf, 
+fs, fsadmin, getConf, loadufs, logLevel, runClass, runTest, runKVTest, runTests, 
+runJournalCrashTest, runMesosTest, readJournal, upgradeJournal, killAll <WORD>, 
+copyDir, clearCache, confDocGen, thriftGen, protoGen, version, validateConf, validateEnv
+
+Some of them are developer side: confDocGen, thriftGen, protoGen
```

Comment:
+1 for including useful commands besides `fs/fsadmin`. For developer commands, perhaps we should just move those tools out of `bin/alluxio` into `dev/alluxio`. I think it should be fine to move these tools since they aren't useful to users anyway. @LuQQiu @apc999 what do you think?

Reply:
would it be better to put in `dev/scripts/alluxio-dev.sh`. I agree that those tools are not useful to most of our users. we could keep `bin/alluxio` as clean as possible.

Reply:
`dev/alluxio` is easier to type and I'm lazy :D. If others prefer `dev/scripts/alluxio-dev.sh` though, I'm fine with that too

Reply:
Changed to 
```
Developer side commands: extensions, confDocGen, thriftGen, protoGen
Should be removed to dev/alluxio and will not be included in this doc.
Other user side commands will all be explained and given usages in this docs.
```